sql/donut.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS members;
+
+CREATE TABLE members (
+    user_id int(11) NOT NULL AUTO_INCREMENT, -- Primary Key
```

Comment:
The number in parentheses isn't really necessary. It's a hint to the number of base-10 digits that is the longest number that can be stored, and doesn't have any special meaning beyond that.